packages/app-store/routing-forms/components/react-awesome-query-builder/config/BasicConfig.ts

```diff
@@ -0,0 +1,453 @@
+// This is taken from "react-awesome-query-builder/lib/config/basic";
```

Comment:
It was earlier imported from `react-awesome-query-builder/lib/config/basic` but now we have copied it here for clarity on the logic that goes in and reduce the size of config as BasicConfig had so many things that we didn't need.


Still keeping some of the configuration commented to make it easier to toggle them back on if needed.